Demos/Blazorise.Demo/Pages/Tests/ToastPage.razor

```diff
@@ -0,0 +1,124 @@
+                            </FieldBody>
+                        </Field>
+                    </Column>
+                    <Column ColumnSize="ColumnSize.IsAuto">
+                        <Field>
+                            <FieldLabel>
+                                Autohide
+                            </FieldLabel>
+                            <FieldBody>
+                                <Switch @bind-Checked="@autohide">Autohide</Switch>
+                            </FieldBody>
+                        </Field>
+                    </Column>
+                    <Column ColumnSize="ColumnSize.IsAuto">
+                        <Field>
+                            <FieldLabel>
+                                Autohide Delay (ms)
+                            </FieldLabel>
+                            <FieldBody>
+                                <NumericEdit @bind-Value="autohideDelay" />
+                            </FieldBody>
+                        </Field>
+                    </Column>
+                </Row>
+            </CardBody>
+        </Card>
+    </Column>
+</Row>
+
+<Row>
+    <Column>
+        <Card Margin="Margin.Is4.OnY">
+            <CardHeader>
+                <CardTitle>Modals (Default)</CardTitle>
+            </CardHeader>
+            <CardBody>
+                <CardText>
+                    A rendered modal with header, body, and set of actions in the footer.
+                </CardText>
+            </CardBody>
+            <CardBody>
+                <Button Color="Color.Primary" Clicked="@(()=>toastProviderRef.Show("Toast Header", "Hello, world! This is a toast message."))">
+                    Show Toast
+                </Button>
+            </CardBody>
+            <CardBody>
+                <CardText>
+                    Show Toasts through service
+                </CardText>
+            </CardBody>
+            <CardBody>
+                <Button Color="Color.Info" Clicked="@(()=>ToastService.Info("This is a toast message from the toast provider.", "Info message"))">
+                    Info
+                </Button>
+                <Button Color="Color.Success" Clicked="@(()=>ToastService.Success("This is a toast message from the toast provider.", "Success message"))">
+                    Success
+                </Button>
+                <Button Color="Color.Warning" Clicked="@(()=>ToastService.Warning("This is a toast message from the toast provider.", "Warning message"))">
+                    Warning
+                </Button>
+                <Button Color="Color.Danger" Clicked="@(()=>ToastService.Error("This is a toast message from the toast provider.", "Error message"))">
+                    Error
+                </Button>
+            </CardBody>
+        </Card>
+    </Column>
+</Row>
+
+<ToastProvider @ref="toastProviderRef" Placement="@toasterPlacement" Animated="@animation" AnimationDuration="@animationDuration" Autohide="@autohide" AutohideDelay="@autohideDelay" />
```

Comment:
It seems like it can when I further checked the files. Please confirm.
Before resolving, please confirm whether this works correctly when you use multiple `ToastProvider` throughout the application?
What's the behaviour if I have both a `ToastProvider` in `App.razor` and in the page itself?

Reply:
Yes, it can be used globally on `App.razor`, and locally by using `ToastProvider` on a page without the service. Here is that use-case in the demo because I want to test both approaches.

Reply:
So if we use both instances at once what would happen? We'll classify that as user error?

Reply:
If there is more than one ToastProvider currently active, and if the user shows a message through the TOastSearvice, then more than one message will display.

Now, if that is an error or not, it is open for discussion....